### PR TITLE
feat: Conversation banner redesign [WPB-1723]

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1723" title="WPB-1723" target="_blank">StoryWPB-1723</a>  Change visual appearance of "guests/externals/services/federated users are present" banner
  </td></table>
  <br />
 

# What's new in this PR?

New designs for "guests/externals/services/federated users are present" banner

### Attachments (Optional)

Screenshot 2024-08-29 at 12 47 21

Screenshot 2024-08-29 at 12 47 34

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/mention/MemberItemToMention.kt [hunk 1]
```diff
@@ -82,7 +82,7 @@ fun PreviewMemberItemToMention() {
         name = "name",
         label = "handle",
         membership = Membership.Federated,
-        searchQuery = "search",
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/mention/MemberItemToMention.kt:85]: I know its just a preview but maybe we keep the searchQuery as it was before?

**Review by alexandreferris — APPROVED** [review]
Nice! Just a small nitpick but not a blocker whatsoever

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/mention/MemberItemToMention.kt (hunk 1 above)
Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/mention/MemberItemToMention.kt:85]: previous version of preview was showing nothing, cause there is no "search" in username, so it hightlight nothing. "na" on the other hand highlight at least something in preview :)

**Review by yamilmedina — APPROVED** [review]
👏

**Pull request merged**